app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -1498,7 +1498,9 @@ $header-font-family: $font-family-sans-serif;
 }
 
 body {
-  background-color: #000;
+  background-image: image-url("main-background.png");
```

Comment:
Properties should be ordered background-attachment, background-image, background-repeat